operator/controller/src/main/java/io/apicurio/registry/operator/feat/KafkaSql.java

```diff
@@ -0,0 +1,25 @@
+package io.apicurio.registry.operator.feat;
+
+import io.apicurio.registry.operator.api.v1.ApicurioRegistry3;
+import io.fabric8.kubernetes.api.model.EnvVar;
+import io.fabric8.kubernetes.api.model.EnvVarBuilder;
+
+import java.util.Map;
+
+import static io.apicurio.registry.operator.resource.app.AppDeploymentResource.addEnvVar;
+import static io.apicurio.registry.operator.utils.Utils.isBlank;
+
+public class KafkaSql {
```

Comment:
This separation is clear and keeps a good readability, happy to follow this style if you prefer it for the Sql datasource :+1:

Reply:
yes, thanks!